agent/php_execute.c

```diff
@@ -71,9 +71,9 @@
  * constructions and teardown as reasons to avoid excessive function calls.
  * This too is erroneous. The cost of calling a function is about 4 assembler
  * instructions. This is negligible. Therefore, as a means of reducing stack
- * usage, if you need stack space it is better to put that usage into a static
- * function and call it from the main function, because then that stack space
- * in genuinely only allocated when needed.
+ * usage, if you need stack space it is better to put that usage into a static*
+ * function and call it from the main function, because then that stack space in
```

Comment:
```suggestion
 * usage, if you need stack space it is better to put that usage into a static
 * function and call it from the main function, because then that stack space is
```